src/git-base.ts

```diff
@@ -58,4 +58,13 @@ export abstract class GitBaseCommand extends Command {
   async cwd() {
     return process.cwd()
   }
+
+  async getCurrentBranchName(options?: SpawnOptions) {
+    let gitCurrentBranchName = await this._spawn('git', ['rev-parse', '--abbrev-ref', '--symbolic-full-name', '@{u}'], options)
+    if (gitCurrentBranchName.stdout.trim() === '@u') {   // catch and counteract bash curly brace evaluation
```

Comment:
can you check for the command exit value(`gitCurrentBranchName.status`) instead or in addition to stdout?

Reply:
Not sure if it handles this specific issue, but, there is a module (cross-spawn) that can help with these types of issue. I would detect the properties of the environment and do the call once instead.

Side note: This is straying from documentation fixes. It's better to open a new issue instead of overloading a story. This helps with velocity, communication and quality.

Reply:
I agree with removing this from this PR and perhaps bring it back separately in another PR

Reply:
I'm happy to back it out, but it is a showstopping nrdk bug. I like the cross-spawn fix (just added that module!), and I'll throw that into a PR now.

Reply:
That's all I need to hear to back adding it as a new ticket to this sprint. Get that credit for the work!

I poked not because this seemed like a good showcase of how we can improve our process. I trust this wouldn't have descended into the black hole of 7 point tickets that take 3 sprints. ;-)